mocks/oppgave-mock/src/main/java/no/nav/oppgave/OppgaveMockImpl.java

```diff
@@ -100,19 +100,20 @@ public Response hentOppgaver(@Context HttpHeaders httpHeaders, @Context UriInfo
                 if (oppgave.hasNonNull(queryKey) && !queryValues.isEmpty()) {
                     possibleMatches.getAndIncrement();
                     JsonNode oppgaveValue = oppgave.get(queryKey);
-                    String firstQueryValue = queryValues.get(0);
-                    switch (oppgaveValue.getNodeType()) {
-                        case STRING:
-                            if (oppgaveValue.asText().equals(firstQueryValue)) matches.getAndIncrement();
-                            break;
-                        case NUMBER:
-                            if (oppgaveValue.asLong() == Long.parseLong(firstQueryValue)) matches.getAndIncrement();
-                            break;
-                        case BOOLEAN:
-                            if (oppgaveValue.asBoolean() == firstQueryValue.equals("true")) matches.getAndIncrement();
-                            break;
-                        default:
-                            throw new IllegalStateException("Ikke støttet NodeType " + oppgaveValue.getNodeType().name());
+                    for (String queryValue : queryValues) {
+                        switch (oppgaveValue.getNodeType()) {
+                            case STRING:
+                                if (oppgaveValue.asText().equals(queryValue)) matches.getAndIncrement();
+                                break;
+                            case NUMBER:
+                                if (oppgaveValue.asLong() == Long.parseLong(queryValue)) matches.getAndIncrement();
+                                break;
+                            case BOOLEAN:
+                                if (oppgaveValue.asBoolean() == queryValue.equals("true")) matches.getAndIncrement();
+                                break;
+                            default:
+                                throw new IllegalStateException("Ikke støttet NodeType " + oppgaveValue.getNodeType().name());
+                        }
                     }
                 }
             });
```

Comment:
Har ikke sjekket nøye men vil linja under 
````            if (matches.get() == possibleMatches.get()) matching.add(oppgave);````
fortsatt funke?

Reply:
Kommer vel an på om aktuell qparam har unike verdier i lista. 
Siden Oppgave bare har single verdier, ingen collections, så bør det være greit å bruke et query med oppgavetype = liste av søkeverdier - en enkel oppgave kan bare matche en av oppgavetypene.

Reply:
Det bør virker greit forutsatt at queryparameterne er unike. Noe usikker over hvor mye tid vi vil investere i dette.
Det virker for vår del som det er.